examples/mistral/readme.md

```diff
@@ -1,8 +1,8 @@
 ## Prerequisites
 * transformers>=4.34.99
 * optimum
+* neural-compressor>=2.4.1
 * ort-nightly
```

Comment:
Will ort-nightly need extra source link? I did not find it from pypi.

Reply:
the following section has the guidance to install it